app/controllers/admin/backups_controller.rb

```diff
@@ -0,0 +1,92 @@
+#-- encoding: UTF-8
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2021 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See docs/COPYRIGHT.rdoc for more details.
+#++
+
+class Admin::BackupsController < ApplicationController
+  include ActionView::Helpers::TagHelper
+
+  layout 'admin'
+
+  before_action :check_enabled
+  before_action :require_admin
+
+  menu_item :backups
+
+  def show
+    @backup_token = Token::Backup.find_by user: current_user
+    last_backup = Backup.last
+
+    if last_backup
+      @job_status_id = last_backup.job_status.job_id
+      @last_backup_date = I18n.localize(last_backup.updated_at)
+      @last_backup_attachment_id = last_backup.attachments.first&.id
+    end
+  end
+
+  def reset_token
+    if request.post?
```

Comment:
I generally don't like the `if request.post?` too much. How about two separate routes for get/post with two different actions?